src/components/Graph.react.js

```diff
@@ -211,6 +223,13 @@ class PlotlyGraph extends Component {
         if (figureChanged) {
             this.plot(nextProps);
         }
+
+        const extendDataChanged =
+            this.props.extendData !== nextProps.extendData;
```

Comment:
Should we also bail out if `extendData` is `null` (or falsy)? I can imagine wanting to say "no updates to this graph", and otherwise you'd have to either
- construct some noop `extendData`, which seems hacky
- or use `preventUpdate`, which also seems hacky but also can't be used in all cases, for example if this is one of several graphs being updated by a multi-output callback.